helper/date_format.go

```diff
@@ -50,14 +52,47 @@ func DateFormatYYYYMMDDNoSlash(s string) string {
 	return template.HTMLEscapeString(t.Format("20060102"))
 }
 
-func DateTimeOnsDatePatternFormat(s string) string {
+func DateTimeOnsDatePatternFormat(s, lang string) string {
```

Comment:
Should we extend/modify this function or create another? Doing so breaks any backwards capability with the library